.github/workflows/tests.yml

```diff
@@ -149,10 +149,13 @@ jobs:
       - name: Run ${{matrix.service}} Shared Tables Tests
         run: _APP_DATABASE_SHARED_TABLES=database_db_main docker compose exec -T appwrite test /usr/src/code/tests/e2e/Services/${{matrix.service}} --debug
 
-  benchamrking:
+  benchmarking:
     name: Benchmark
     runs-on: ubuntu-latest
     needs: setup
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
Without this, this fails in repositories that have restricted permissions for workflows:
```yaml
      - name: Comment on PR
        uses: peter-evans/create-or-update-comment@v4
```